src/xcode/ENA/ENA/Source/Workers/SQLiteKeyValueStore.swift

```diff
@@ -151,12 +175,27 @@ class SQLiteKeyValueStore {
 			try db.executeUpdate(deleteStmt, values: [])
 			try db.executeUpdate("VACUUM", values: [])
 			log(message: "Flushed SecureStore", level: .info)
+			rekeyDatabase()
```

Comment:
Why do we not just wipe everything on reset? Even the URL overrides? I mean reset should just outright reset everything... shouldn't it? Resetting to be means to just close the database, if open and then delete the database files... This would also remove the need for a rekey...

Reply:
That is true. I added a deleteDatabase() Function to just remove the database files. So no more rekey is needed

Reply:
I will approve this - however I think you will leave data behind. (transaction log files).
They are encrypted but will cause issues when you re-create the database again.
Let's have a cal about this tomorrow - eh today if you don't mind. I can show you how to tell SQLite to delete the file for you.